heliopy/data/util.py

```diff
@@ -128,6 +130,9 @@ def processing_func(local_dir, local_fname, **processing_kwargs)
             try:
                 df = processing_func(local_dir, fname + extension,
                                      **processing_kwargs)
+                if units_func is not None:
+                    units = units_func(local_dir, fname + extension,
+                                       **processing_kwargs)
```

Comment:
Not sure about the **processing_kwargs, but since this is for reading the CDF files first, I used the processing_func as it was.